paper/paper.md

```diff
@@ -96,16 +100,13 @@ teaching workshops with similar learning objectives.
 The University of Michigan instance of the Carpentries has taught workshops
 using the curriculum created by the Carpentries organization since 2015.
 In that time, our instructors have found several advantages and disadvantages to
-using the Carpentries curriculum.
-Some advantages that we found are:
-
-1. It is modular, so workshop organizers can choose any programming language
-lesson (e.g. R or Python) to pair with lessons on the Unix shell and version
-control;
-1. It has been refined over years by many contributors and taught at workshops
-around the world; and
-1. It demonstrates good pedagogy for teaching beginner data science
-practitioners.
+using the Software Carpentry curriculum in particular.
+Some advantages are that any programming language lesson (e.g. R or
+Python) can be paired with lessons on the Unix shell and version control;
+the lessons have been refined over years by many contributors and taught at
+workshops around the world; and
+the instructional design demonstrates good pedagogy for teaching beginner data
+science practitioners.
 
 However, we found that the curriculum did not meet all of our needs.
 The disadvantages that we found are:
```

Comment:
Should we keep this as a numbered list, or collapse into a paragraph?

I think I like it as a list but we do have several of them in the paper already.

Reply:
And maybe we should add something in about it being more of a reference than a lesson plan, as Pat mentioned